### PR TITLE
Add CNVM to our Readme

### PR DESCRIPTION
**Pull request by oren-zohar**
### Summary of your changes
Readme update to include CNVM

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,39 +4,38 @@
 
 # Cloudbeat
 
-Cloudbeat analyzes cloud assets for security compliance and sends findings to Elasticsearch as part of
-the [Cloud Security Posture](https://www.elastic.co/blog/secure-your-cloud-with-elastic-security) plugin in Kibana.
+Cloudbeat is a tool that analyzes cloud assets for security compliance and sends findings to Elasticsearch.
+It is designed to be used as part of the [Cloud Security Posture](https://www.elastic.co/blog/secure-your-cloud-with-elastic-security) plugin in Kibana.
```
Comment (amirbenun) [inline README.md:8]:
```suggestion
It is designed to be used as part of the [Cloud Security](https://www.elastic.co/blog/secure-your-cloud-with-elastic-security) plugin in Kibana.
```

File: README.md [hunk 2]
```diff
@@ -4,39 +4,38 @@
...
+
+### CSP Security Policies
+
+Cloudbeat uses security policies from the [CSP Security Policies](https://github.com/elastic/csp-security-policies) repository to evaluate cloud resources.
 
 ## Getting Started
 
-To get started with Cloud Security Posture on your cluster, see
-our [documentation](https://www.elastic.co/guide/en/security/master/get-started-with-kspm.html#kspm-setup-unmanaged).
+To get started with Cloud Security Posture on your cluster, please refer to our documentation:
```
Comment (amirbenun) [inline README.md:17]:
```suggestion
To get started with Cloud Security on your cluster, please refer to our documentation:
```

File: tests/README.md [hunk 3]
```diff
@@ -179,28 +179,36 @@ For example `EC2` data cases will be located under `aws_ec2_test_cases.py`.
...
+
 ```python
 cis_aws_log_cases = {
     **cis_aws_log_3_1,
     ...
 ```
-- If just adding a new test case to existing test suite, no additional steps are required, the case will be added automatically
+
+- If just adding a new test case to existing test suite, no additional steps are required, the case will be added
+  automatically
```
Comment (amirbenun) [inline tests/README.md:208]: revert

File: tests/README.md [hunk 4]
```diff
@@ -134,15 +130,18 @@ The project main folders are:
...
+
 ```shell
 from loguru import logger
 
 logger.info("Start logging")
 ```
 
-Basic logging configuration is realized through [environment variables](https://github.com/Delgan/loguru/blob/master/loguru/_defaults.py)
+Basic logging configuration is realized
+through [environment variables](https://github.com/Delgan/loguru/blob/master/loguru/_defaults.py)
```
Comment (amirbenun) [inline tests/README.md:141]: revert

File: tests/README.md [hunk 5]
```diff
@@ -228,7 +239,9 @@ markers = [
     ... # <-- add new marker
 ```
 
-- Execute the test suite by running the following command and replacing marker `aws_logging_rules` with newly defined marker
+- Execute the test suite by running the following command and replacing marker `aws_logging_rules` with newly defined
+  marker
```
Comment (amirbenun) [inline tests/README.md:243]: revert

Comment (oren-zohar) [reply tests/README.md:243]: this is automatically formatted by our pre-commit hook to enforce line length (it won't affect the actually markdown output)

File: tests/README.md (hunk 4 above)
Comment (oren-zohar) [reply tests/README.md:141]: this is automatically formatted by our pre-commit hook to enforce line length (it won't affect the actually markdown output)

File: tests/README.md (hunk 3 above)
Comment (oren-zohar) [reply tests/README.md:208]: this is automatically formatted by our pre-commit hook to enforce line length (it won't affect the actual markdown output)

**Review by amirbenun — APPROVED** [review]

**Review by orestisfl — APPROVED** [review]

**Review by olegsu — APPROVED** [review]

**Pull request merged**